stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -377,6 +377,33 @@ julia> lmul!(F.Q, B)
 """
 lmul!(A, B)
 
+# We may inline the matmul2x2! and matmul3x3! calls for `α == true`
+# to simplify the @stable_muladdmul branches
+function matmul2x2or3x3_nonzeroalpha!(C, tA, tB, A, B, α, β)
+    if size(C) == size(A) == size(B) == (2,2)
+        matmul2x2!(C, tA, tB, A, B, α, β)
+        return true
+    end
+    if size(C) == size(A) == size(B) == (3,3)
+        matmul3x3!(C, tA, tB, A, B, α, β)
+        return true
+    end
+    return false
+end
+function matmul2x2or3x3_nonzeroalpha!(C, tA, tB, A, B, α::Bool, β)
```

Comment:
```suggestion
function matmul2x2or3x3_nonzeroalpha!(C, tA, tB, A, B, ::Bool, β)
```

Reply:
I did consider this, but I wondered if having the variable name available might add context (e.g., when inspecting the code with `Cthulhu` etc.). One option might be to add a comment.

Reply:
I just saw that the variable wasn't used, so this is "better style". But keeping it for code inspection uses is also fine.